_pages/mobile-interview-project.md

```diff
@@ -8,40 +8,55 @@ header:
 
 # Hello!
 
-*This site outlines the info and guidelines you’ll need to complete Square’s Mobile Take-Home Project: An Employee Directory App. You’ll have a week to complete the question – the actual engineering work should take about 3-4 hours, and you can split this across the week however you see fit (all at once, over a few days, etc). If you have any questions during the process please share them with your recruiting contact and we’ll get back to you ASAP. Good luck!*
+Thank you for choosing to interview at Square! We are so excited for you and we hope that you have a great experience with us.
+
+This site outlines the info and guidelines you’ll need to complete Square’s Mobile Take-Home Project: an employee directory app.
+
+You’ll have a week to complete the question – the actual engineering work should take about 3-4 hours, and you can split this across the week however you see fit (all at once, over a few days, etc). If you have any questions during the process, please share them with your recruiting contact and we’ll get back to you ASAP.
+
+Good luck!
 
 # Overview
-Build an employee directory app that shows a list of employees parsed from the provided JSON endpoint.
 
-Take care to properly handle errors returned from the endpoint (or other network errors like timeouts), and ensure you do not waste network bandwidth – load expensive resources such as large photos on-demand only. Photos at a given URL will never change. Once one is loaded, you do not need to reload the photo. If an employee’s photo changes, they will be given a new photo URL (eg, you can treat the URL as a GUID).
+Build an employee directory app that shows a list of employees from the provided endpoint.
```

Comment:
endpoint -> endpoints

Reply:
Though i guess this is fine if we consider the other endpoints just for testing!